crates/sozo/ops/src/auth.rs

```diff
@@ -154,7 +160,8 @@ where
 
     for new_owner in new_owners {
         let resource_selector =
-            get_resource_selector(ui, world, &new_owner.resource, default_namespace).await?;
+            get_resource_selector(ui, world, &new_owner.resource.clone(), default_namespace)
```

Comment:
We shouldn't need to clone if we pass a ref.

```suggestion
            get_resource_selector(ui, world, &new_owner.resource, default_namespace)
```